### ISSUE TITLE
Ability to view comments

### ISSUE THREAD

**Issue by imhemish** [vixalien/muzika#170]
A lot of album songs and video songs have comments under them, maybe we can have the ability to view comments, but protruding out a separate AdwDialog (which will transform into bottom sheet), like on Android. It can be an optional feature which can be enabled in Preferences.

Probably this would require additions in muse

**Comment by vixalien** [comment]
> Probably this would require additions in muse

Correct. But to be honest I have to agree that this is not a high-priority feature for me.

**Comment by vixalien** [comment]
This feature is not available in the browser, so there's not much I can do at the moment.
